app/js/app/directives/react_classes/UnknownBlock.js

```diff
@@ -36,17 +36,19 @@ define(["react", "jquery"], function(React,$) {
 			render: function() {
 				return (
 					<Block type="unknown" blockTypeTitle="?" doc={this.props.doc} onChange={this.onDocChange}>
-						<div className="row">
+						<div className="row">{this.props.doc}
 							<div className="large-8 large-offset-2 columns text-center">
 								Please choose a block type: <br/>
+								<strong>Careful:</strong> if you are creating a quiz, please only choose a <strong>quiz section</strong> at the top level <br />
```

Comment:
I'm not sure this is necessary, we don't have a warning to to create tabs inside accordions etc, in most cases the team won't be making quizzes. Perhaps just something strongly worded the the affect within the wiki / point both content teams towards it?

Reply:
@jsharkey13 @mlt47 opinions?

Reply:
Well, the vast majority of the time it won't be relevant so I'd err toward not saying anything unless it's actively an error (which this can't easily check).